includes/admin/class-coblocks-crop-settings.php

```diff
@@ -19,8 +19,24 @@ class CoBlocks_Crop_Settings {
 
 	const CROP_META_KEY = 'crop-image-data';
 
+	/**
+	 * This class's instance.
+	 *
+	 * @var CoBlocks
+	 * @since 1.14.0
+	 */
 	private static $instance;
 
+	/**
+	 * Main CoBlocks Instance.
```

Comment:
This should be updated, and the description below as well.

Reply:
Fixed.